src/guides/reset_with_sd_card.html

```diff
@@ -0,0 +1,60 @@
+---
+layout: troubleshooting-guide
+description: Home Assistant Green
+guideName: Resetting the device using an SD card
+tools:
+  title: Prerequisites
+  image: green_sd_card.png
+  content: |
+    <ul>
+        <li>Home&nbsp;Assistant Green</li>
+        <li>Home Assistant 13 or older and you cannot upgrade to Home Assistant 13.1 or later</li>
+        <li>Micro SD card</li>
```

Comment:
I think officially (as per SD Association) the name is `microSD`.

```suggestion
        <li>microSD card</li>
```